src/sentry/api/endpoints/organization_projects.py

```diff
@@ -97,6 +97,8 @@ def get(self, request, organization):
                 elif key == "!team":
                     team_list = list(Team.objects.filter(slug__in=value))
                     queryset = queryset.exclude(teams__in=team_list)
+                elif key == "user_projects":
```

Comment:
what about like `is_member` or `user_is_member` or `has_membership`?

Reply:
i'll switch over to is_member